src/main/java/com/stripe/model/Account.java

```diff
@@ -181,11 +184,16 @@ public CapabilityCollection capabilities(Map<String, Object> params) throws Stri
   public CapabilityCollection capabilities(Map<String, Object> params, RequestOptions options)
       throws StripeException {
     String url =
```

Comment:
nit: if we're changing this parameter name from url to path on [StripeResponseGetter.request](https://github.com/stripe/stripe-java/pull/1596/files#diff-55b4e268c6f5bded4a1a2dbe940b86c9e22f6ef986a54e6a053041b4284a1e10R14), change the name of this variable too for consistency?

Reply:
Done.